packages/react-components/react-message-bar/library/src/contexts/messageBarBodyContext.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+export type MessageBarBodyContextValue = {
+  inlineLinks: boolean;
+};
+
+const messageBarContext = React.createContext<MessageBarBodyContextValue | undefined>(undefined);
+
+export const messageBarBodyContextDefaultValue: MessageBarBodyContextValue = {
+  inlineLinks: false,
+};
+
+export const MessageBarBodyContextProvider = messageBarContext.Provider;
+export const useMessageBarBodyContext = () => React.useContext(messageBarContext) ?? messageBarBodyContextDefaultValue;
```

Comment:
It should be inverted i.e. `react-link` has a context provider as in future another component can also tweak `Link`'s behavior

Reply:
right, makes sense, updated 👍🏻